frontend/index.html

```diff
@@ -15,11 +15,16 @@
     />
     <meta
       property="og:image"
-      content="https://fittoring.com/static/og-image.png"
+      content="https://www.fittoring.com/static/og-image.png"
     />
-    <meta property="og:url" content="https://fittoring.com/" />
+    <meta property="og:url" content="https://www.fittoring.com/" />
     <meta property="og:type" content="website" />
 
+    <meta
+      name="naver-site-verification"
+      content="42d897e130c99bef55e694c2dd9bb7755ddb9302"
+    />
+
     <title>핏토링</title>
     <link rel="icon" href="/favicon-logo.svg" />
     <link rel="canonical" href="https://www.example.com/original-page-url" />
```

Comment:
p4: 캐노니컬 태그 설정 추가하면 좋을 것 같아 👍
```suggestion
    <link rel="canonical" href="https://www.fittoring.com" />
```